bikeshed/boilerplate/whatwg/header-RD.include

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<meta name="theme-color" content="#3c790a">
```

Comment:
Maybe we shouldn't do this, given they are generally unstyled? Same for the favicon.

Reply:
I think there's a difference between styling and branding; I think it makes sense to keep the branding?

Reply:
I think I'd prefer it if we used a different favicon (and maybe color if we want to keep that) for these so you can tell from your tabs which are Living and which are stale.

Reply:
We should also consider that for snapshots I suppose.

Reply:
Good idea! Maybe grayed out? Or more extreme, upside down and inverted?

Reply:
I like grayed out.

Reply:
Updated (depends on https://github.com/whatwg/whatwg.org/pull/204). I'd like to do this for commit snapshots too but that's a tiny bit trickier and we already have a lot of things in flight. Let me file a meta issue to remind us and we can do it later.